main.py

```diff
@@ -1,15 +1,77 @@
-from typing import Union
-
 from fastapi import FastAPI
+from pyathena import connect
+from typing import Union
+import pandas as pd
+import base64
+import boto3
+import redis
 
 app = FastAPI()
+r = redis.Redis(host='localhost', port=6379, db=0)
+athena_client = boto3.client('athena')
+s3_client = boto3.client('s3')
+
+AWS_DATA_CATALOG = "AwsDataCatalog"
+AWS_SCHEMA_DATABASE_NAME = "ensembl-parquet-meta-schema"
+AWS_S3_OUTPUT_DIR = "s3://ensembl-athena-results/"
+
+@app.get("/api/v1")
+def root():
+    return "Ensembl's data lakehouse backend"
+
+
+@app.get("/api/v1/data")
```

Comment:
Following first comment this one should become
```suggestion
@app.get("/data")
```